config.js

```diff
@@ -50,6 +51,7 @@ var environment = {
   awsRegion: process.env.AWS_REGION,
   sendgridApiKey: process.env.SENDGRID_API_KEY,
   sendgridFrom: process.env.SENDGRID_FROM,
+  gdalTranslateBin: process.env.GDAL_TRANSLATE_BIN,
```

Comment:
If you can't set `PATH`, you won't be able to set `GDAL_TRANSLATE_BIN` either...

Re: pre-emptively failing when environment variables are missing, I've been using [`require-env`](https://www.npmjs.com/package/require-env) for a long time now.